OpenSim/Tools/CMCTool.cpp

```diff
@@ -86,6 +87,35 @@ CMCTool::CMCTool() :
 }
 //_____________________________________________________________________________
 /**
+ * Default constructor with a specified number of threads (jobs).
+ */
+CMCTool::CMCTool(int numThreads) :
+    AbstractTool(),
```

Comment:
A one-argument constructor should always be `explicit` unless you intended it as an implicit conversion. I'm guessing implicitly converting int to CMCTool is not what you had in mind!